syntax = "proto3";

service FaceService {
  rpc DetectFacePose (DetectFaceRequest) returns (DetectFaceResponse);
}

// SERVICE DETECT FACE

message DetectFaceRequest {
  string image_path = 1;
}

message DetectFaceResponse {
  string result = 1;
}
